paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1083,6 +988,43 @@ def start_slo_watcher_threads(self) -> None:
             sfx_api_token=load_system_paasta_config().get_monitoring_config()['signalfx_api_key'],
         )
 
+    def get_button_element(self, button, is_active):
```

Comment:
I think the SlackDeployProcess could reuse a lot of this, if MFD just defines the button texts.